### DISCUSSION TITLE
Hybrid gpt/mbr Ventoy install?

### DISCUSSION THREAD

**Discussion by chriscjs** [ventoy/Ventoy#2213] [General]
Both the Windows and  linux distros  install images are able to boot from both bios/mbr and uefi/gpt so I am surprised this is not part of Ventoy itself.   This would remove the need to choose between mbr or gpt when installing Ventoy to the usb drive.

**Comment by meeuw** [comment]
I was wondering the same thing and did some research. I think Windows and Linux combine CD/DVD file systems (UDF/ISO 9660) with GPT as an hybrid.

The downside of these CD/DVD file systems is that they are read/append only (WORM) and there is no way to add/remove ISO/WIM/IMG/VHD(x)/EFI files to your USB drive without using a specialized tool (imagine you need to run ventoy2disk for each ISO file and you're only allowed to add new files).
